### DISCUSSION TITLE
Welcome to powercat-creator-kit Discussions!

### DISCUSSION THREAD

**Discussion by slaouist** [microsoft/powercat-creator-kit#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JonasInduce** [comment]
Command bar is very useful but I want to resize the icons.
Is it possible to resize the icon or text on the command bar.
And how to use input event on command bar.

**Reply by scottdurow** [reply]
Hi @JonasInduce - Thank you for your feedback on the CommandBar - I am really happy to hear you are finding it useful 👍 
At this time the size of the text and icons are fixed to be in line with the [Fluent UI style guide and the other controls](https://developer.microsoft.com/en-us/fluentui#/styles/web/typography#sizes).
Could I ask more about what level of control you would like over the icon and text size, please?

It's worth noting that if you are seeing the flyout and command bar buttons being different sizes, ensure you have the **Scale to fit** setting turned off under **Settings** -> **Display**

The `InputEvent` property is used to programmatically set the focus on the command bar - see https://github.com/microsoft/powercat-code-components/tree/main/CommandBar#setting-focus-on-the-control.

**Comment by JonasInduce** [comment]
Hi Scott. Thanks for the reply.
Right now I just miss being able to resize icon and text.
It is in the mobile app with setting Scale to fit disabled and iPhone 13 Pro that the icons become really small.
Think it's 10 px while everything in Gallery and other text can be changed to any size.

Sincerely Jonas Linderos

Från: Scott Durow ***@***.***>
Datum: tisdag, 28 juni 2022 03:31
Till: microsoft/powercat-creator-kit ***@***.***>
Kopia: Jonas Linderos ***@***.***>, Mention ***@***.***>
Ämne: Re: [microsoft/powercat-creator-kit] Welcome to powercat-creator-kit Discussions! (Discussion #15)

Hi @JonasInduce<https://github.com/JonasInduce> - Thank you for your feedback on the CommandBar - I am really happy to hear you are finding it useful 👍
At this time the size of the text and icons are fixed to be in line with the Fluent UI style guide and the other controls<https://developer.microsoft.com/en-us/fluentui#/styles/web/typography#sizes>.
Could I ask more about what level of control you would like over the icon and text size, please?

It's worth noting that if you are seeing the flyout and command bar buttons being different sizes, ensure you have the Scale to fit setting turned off under Settings -> Display

The InputEvent property is used to programmatically set the focus on the command bar - see https://github.com/microsoft/powercat-code-components/tree/main/CommandBar#setting-focus-on-the-control.

**Reply by scottdurow** [reply]
Hi @JonasInduce - thanks for the update. Do you think you would need control over both the flyout text/icon size and the main command bar - or just the main command bar icon/text size?

**Comment by JonasInduce** [comment]
hi i think a setting for size of both icon and text is enough. /Jonas

Hämta Outlook för iOS<https://aka.ms/o0ukef>
________________________________
Från: Scott Durow ***@***.***>
Skickat: Wednesday, June 29, 2022 2:38:50 AM
Till: microsoft/powercat-creator-kit ***@***.***>
Kopia: Jonas Linderos ***@***.***>; Mention ***@***.***>
Ämne: Re: [microsoft/powercat-creator-kit] Welcome to powercat-creator-kit Discussions! (Discussion #15)


Hi @JonasInduce<https://github.com/JonasInduce> - thanks for the update. Do you think you would need control over both the flyout text/icon size and the main command bar - or just the main command bar icon/text size?
